### ISSUE TITLE
Grafische Lösung: Ansprechpartner...

### ISSUE THREAD

**Issue by marcus-herrmann** [clarat-org/clarat#517]
...gemeinsame und einzelne Kontaktdaten. Grafisches Brainstorming Desktop (die Vielzahl der Ansprechpartner tritt natürlich nicht gleichzeitig auf, die 3 Container sind 3 Varianten):

![offer show](https://cloud.githubusercontent.com/assets/505181/10659327/49e02c3c-789f-11e5-82b7-73e8a1c932e9.png)

**Comment by Twiek** [comment]
@jjungle: what do you think?

**Comment by Twiek** [comment]
@jjungle Bitte noch mal diskutieren wie viele Infos hier wirklich angezeigt werden müssen. Dem User 10 mehr oder minder identische Kontaktmöglichkeiten zu präsentieren schadet mehr als das es nutzt.

**Comment by jjungle** [comment]
Wir haben uns hier jetzt auf max. 3 Ansprechpartner verständigt.
http://newsticker.clarat.org/?p=2168

**Comment by Twiek** [comment]
Note to self: Dann auch Adressen für Briefangebote mitdenken.

**Comment by Twiek** [comment]
Zitat aus besagtem Newsticker: "Wir geben höchstens drei Ansprechpartner mit jeweils maximal zwei Telefonnummern an."

**Comment by KonstantinKo** [comment]
Prototyp ist soweit umgesetzt, aber:

- **funktioniert noch nicht im arabischen rtl layout**
- Was passiert, wenn wir für einen Kontakt keinen Vornamen, Nachnamen, oder "operational name" haben?
- Der Prototyp zeigt eine Adressausgabe auf der rechten Seite, aber Adressen werden für Kontaktpersonen bisher noch nicht angezeigt. Dafür braucht es noch Logik: Wann und wie Adressen anzeigen?
- Zuständigkeit gibt es bisher auch noch nicht. Wie funktioniert das?

**Comment by marcus-herrmann** [comment]
> * funktioniert noch nicht im arabischen rtl layout

Fix hierfür in Commit 5b9920580ebb323fac26644236fdf925c591b740

> * Was passiert, wenn wir für einen Kontakt keinen Vornamen, Nachnamen, oder "operational name" haben?

Weiß nicht ganz, ob die Frage an mich gerichtet ist, aber kann man für diesen Fall nicht den String "Ansprechpartner bei ${orga.name}" ausgeben?

> * Zuständigkeit gibt es bisher auch noch nicht. Wie funktioniert das?

Grafisch würde ich die Nennung der Zuständigkeit in der Subline (kleinere Schrift unter Namen) sehen.

**Comment by Twiek** [comment]
@marcus-herrmann: Stimme Dir was die Nennung der Zuständigkeit angeht zu.
Wegen der unklaren Frage bitte Konstantin kontaktieren.

Hast Du ansonsten einen Vorschlag bezüglich der Adressausgabe. Geht hier z.B. um 'Briefangebote' zu denen man nicht hingehen kann (daher keine Anzeige in der rechten Seitenleiste), denen man aber einen Brief schreiben kann.

**Comment by marcus-herrmann** [comment]
> Hast Du ansonsten einen Vorschlag bezüglich der Adressausgabe. Geht hier z.B. um 'Briefangebote' zu denen man nicht hingehen kann (daher keine Anzeige in der rechten Seitenleiste), denen man aber einen Brief schreiben kann.

Ich würde die Adresse in der rechten Seitenleiste lassen, einer "Postfach-Adresse" aber einen Container geben, mit Alarm-Charakter, zB durch subtile Querstreifen ("Absperrband"/"Crime Scene"), sodass das Auge hängen bleibt. Eventuell noch flankiert mit einem kleinen Label à la "Du kannst diese Adresse nicht aufsuchen, aber einen Brief dahin schreiben".

**Comment by marcus-herrmann** [comment]
>> Was passiert, wenn wir für einen Kontakt keinen Vornamen, Nachnamen, oder "operational name" haben?

> Weiß nicht ganz, ob die Frage an mich gerichtet ist, aber kann man für diesen Fall nicht den String "Ansprechpartner bei ${orga.name}" ausgeben?

@KonstantinKo -  Technisch steht dem vermutlich nicht viel im Weg, aber wäre so etwas überhaupt in Eurem Sinne?

**Comment by JulianJung** [comment]
Kurze Ergänzung dazu: Ein Kontakt ohne Vorname, Nachname oder Operational Name, darf immer nur dann einem Angebot zugeordnet sein, wenn es der einzige Kontakt dieses Angebots ist.

**Comment by Twiek** [comment]
@marcus-herrmann Würde die Adresse bei Briefangeboten nur ungern in der Seitenleiste anzeigen, insofern bitte mal prüfen wie man es in der 'Wo kannst du dich melden? Box anzeigen kann.

Die "Ansprechpartner bei ${orga.name}"  Lösung für Kontakte bei denen wir keinen Vornamen, Nachnamen, oder "operational name" haben finde ich gut.

Ansonsten bitte darauf achten, dass immer nur eine Person assigned ist (man muss sich inzwischen leider selber manuell wieder unassignen, auch wenn man jemanden anderes zuweist).

**Comment by marcus-herrmann** [comment]
@Twiek Dachtest du an so etwas? Eventuell noch mit einem "Wasserzeichen" (Kouvert-Illustration) im weißen Container?

<img width="1275" alt="bildschirmfoto 2016-08-11 um 14 51 34" src="https://cloud.githubusercontent.com/assets/505181/17589124/3307998e-5fd3-11e6-9324-4eabfc86d847.png">

**Comment by Twiek** [comment]
Nein, ich meinte es im 'Wo kannst du dich melden?' Feld und nicht gesondert hervorgehoben. Lass am besten noch mal drüber sprechen.

**Comment by marcus-herrmann** [comment]
So wie ich dich verstanden habe, ist das dann keine grafische Frage mehr.
